code/network/multiteamselect.cpp

```diff
@@ -1242,12 +1242,31 @@ void multi_ts_blit_wings()
 	}	
 }
 
+bool multi_ts_is_player_in_slot(int slot_idx)
+{
+	Assertion(slot_idx > 0, "Call to check if player is allowed in a negative number slot!");
+	Assertion(slot_idx < MAX_WSS_SLOTS, "Call to check if player is allowed in slot greater than Max Slots!");
+
+	return (Multi_ts_team[Net_player->p_info.team].multi_ts_player[slot_idx] != nullptr);
+}
+
+bool multi_ts_is_player_allowed_in_slot(int slot_idx)
+{
+	Assertion(slot_idx > 0, "Call to check if player is allowed in a negative number slot!");
```

Comment:
Should be `>=` there.

And don't forget the `Net_player != nullptr` check too.